spec/controllers/users_controller_spec.rb

```diff
@@ -34,6 +34,22 @@
       expect(flash[:alert]).not_to be_nil
       expect(response).to redirect_to(root_path)
     end
+
+    context 'if the user has an unfilled name' do
+      render_views
+
+      it 'clears the name input and sets it to error' do
+        allow(@controller).to receive_messages(current_user: user)
+        user.update(name: User::EMPTY_NAME)
+
+        get :edit, params: { id: user }
+
+        doc = Nokogiri::HTML(response.body)
```

Comment:
there is probably some matcher for this kind of stuff, but i dont remember, so whatever.